main.go

```diff
@@ -24,16 +24,38 @@ var Version = "unknown"
 const (
 	msgCOC        = "Please find our Code Of Conduct here: https://bcneng.github.io/coc/"
 	msgNetiquette = "Please find our Netiquette here: https://bcneng.github.io/netiquette/"
+	msgWelcome    = `*Welcome to BcnEng's Slack!*
+
+BcnEng’s mission is to let Barcelona’s tech hub to become one of the best tech communities around the globe.
+
+How?
+
+- Promoting and spreading technical knowledge
+- Enhancing personal and professional relationships in a frame of diversity
+- Foment training among the community
+
+Thanks for read and accept our <https://bcneng.org/coc|Code Of Conduct>. Please *take your time* reading our <https://bcneng.org/netiquette|Netiquette> as well.
+
+We as members, contributors, and admins pledge to make participation in our community a harassment-free experience for everyone, 
+regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity 
+and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, 
+or sexual identity and orientation.
+
+We invite you to take a look to our channels, to enjoy and *have fun*. And why not? Say hello to everyone in #general.
```

Comment:
```suggestion
We invite you to take a look at our channels, to enjoy and *have fun*. And why not? Say hello to everyone in #general.

Remember that BcnEng has a lot of members, so try to be specific and choose the appropriate channels for your messages. Finally, take a look at the topics of the channels before posting, because some of them have specific rules about content and format.
```